package/recalbox-system/recalbox-system.mk

```diff
@@ -9,21 +9,13 @@ RECALBOX_SYSTEM_SOURCE=
 RECALBOX_SYSTEM_VERSION=0.2
 
 ifeq ($(BR2_cortex_a7),y)
-	RECALBOX_SYSTEM_RPI_VERSION=rpi2-unstable
+	RECALBOX_SYSTEM_RPI_VERSION=rpi2
```

Comment:
maybe ;-)
i know you would change that because i guess that the BR_FINAL can be
removed too.
So, do as you wish ;-)

please note that the wiki is to upgrade because i used the name you gave me
for recalbox.conf + the wiki is missing the /rpi1 or /rpi2 path.

Nicolas

2016-01-28 8:18 GMT+01:00 digitalLumberjack <notifications@github.com>:

> In package/recalbox-system/recalbox-system.mk
> <https://github.com/recalbox/recalbox-buildroot/pull/155#discussion_r51087575>
> :
>
> > @@ -9,21 +9,13 @@ RECALBOX_SYSTEM_SOURCE=
> >  RECALBOX_SYSTEM_VERSION=0.2
> >
> >  ifeq ($(BR2_cortex_a7),y)
> > -	RECALBOX_SYSTEM_RPI_VERSION=rpi2-unstable
> > +	RECALBOX_SYSTEM_RPI_VERSION=rpi2
>
> maybe we could use only one variable ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/recalbox/recalbox-buildroot/pull/155/files#r51087575>.
>